pkg/pillar/containerd/containerd.go

```diff
@@ -525,6 +539,40 @@ func (client *Client) CtrListTaskIds(ctx context.Context) ([]string, error) {
 	return res, nil
 }
 
+func (client *Client) CtrNewContainer(ctx context.Context, name string, containerImage containerd.Image) (containerd.Container, error) {
+	var spec specs.Spec
+
+	spec.Root = &specs.Root{
+		Readonly: false,
+	}
+
+	mount := specs.Mount{
+		Destination: "/persist",
```

Comment:
What is the reason to force this bind mount by default?

Reply:
It is used only by code from this PR and there it is needed.
Rename to `CtrNewContainerWithPersist`? Or do you want that I add a parameter to the method?

Reply:
I think you could pass the spec as parameter, so it would give more flexibility to the function and can be used in the future by non bpftrace related stuff....

Reply:
@rene that one already exists here: https://github.com/lf-edge/eve/pull/4150/files#diff-522c973a554c295e1ef28879a4634f6423a3e5a2f65def9335567e34476c7f50R543 ?